draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.tsx

```diff
@@ -130,7 +130,7 @@ export class Collapsible extends React.Component<CollapsibleProps, State> {
           )}
           <div>
             <IconButton
-              label="Toggle section"
+              label={title}
```

Comment:
Would this be a negative experience for someone who is tabbing through the UI instead? ie. it might read the title of the collapsible and then the title again when they get to the button.

Would a `labelledBy` be more appropriate? Or present the same issue?